data/specials.inc

```diff
@@ -536,3 +536,4 @@ gSpecials::
 	def_special TrySetBattleTowerLinkType
 	def_special SetTotemBoost
 	def_special TrySpecialOverworldEvo
+    def_special TrySkyBattle
```

Comment:
Please replace the empty spaces with tabs. Egg asked you to do it twice and you marked those conversations as resolved even though you're still not complying.

Reply:
okay I usually do tabs, so I thought Egg's first comment was 

> hey replace tabs with spaces here

and then when I got the comment again I thought

> oh I must not have done it before, whoops

In my defense I don't think

> tabs

is a very helpful PR comment. Thanks for elaborating

but I swear this time I did tabs: https://i.imgur.com/Fk95PYe.gif